Grace.py

```diff
@@ -0,0 +1,56 @@
+# Task 1 - parse a greeting and a name

```

Comment:
Please split it into 3 files, e.g. `grace_1.py`, `grace_2.py` and `grace_3.py`. Also, please start the filename with lowercase.